File: Leetcode/PY/Leetcode1078-Bigram分词.py
class Solution:
    def findOcurrences(self, text: str, first: str, second: str) -> List[str]:
        rec = text.split()
        ans = []
        for i in range(len(rec)):
            if rec[i] == first and i + 2 < len(rec):
                if rec[i + 1] == second: ans.append(rec[i + 2])
        return ans

File: Leetcode/JS/Leetcode1078-Bigram分词.js
var findOcurrences = function(text, first, second) {
    const rec = text.split(" ");
    const ans = [];
    for(let i = 0;i < rec.length;i++){
        if(rec[i] == first && i + 2 < rec.length){
            if(rec[i + 1] == second) ans.push(rec[i + 2]);
        }
    }
    return ans;
};